test/e2e/performanceprofile/functests/6_mustgather_testing/mustgather.go

```diff
@@ -39,60 +39,68 @@ var _ = Describe("[rfe_id: 50649] Performance Addon Operator Must Gather", func(
 		if mustGatherFailed {
 			Skip("No mustgather directory provided, check MUSTGATHER_DIR environment variable")
 		} else {
+
 			mustgatherPathContent, err := ioutil.ReadDir(mustGatherPath)
+			Expect(err).To(BeNil(), "failed to read the Mustgather Directory %s: %v", mustGatherPath, err)
+
+			for _, directory := range mustgatherPathContent {
+				if strings.Contains(directory.Name(), "must-gather") {
+					mustGatherContentDir = filepath.Join(mustGatherPath, directory.Name())
+				}
+			}
+
+			// local test folder, test node
+			mustGatherTarDir = filepath.Join(mustGatherContentDir, "nodes", "worker1")
+
 			Expect(err).To(BeNil(), "failed to read the Mustgather Directory %s: %v", mustGatherPath, err)
 			for _, file := range mustgatherPathContent {
-				if strings.Contains(file.Name(), "registry") {
-					mustGatherContentDir = filepath.Join(mustGatherPath, file.Name())
+				if strings.Contains(file.Name(), "sysinfo.tgz") {
+					mustGatherTgz = filepath.Join(mustGatherPath, file.Name())
 				}
 			}
+			Expect(strings.Contains(mustGatherTgz, "sysinfo.tgz"))
+			//untar tgz
```

Comment:
does this comment mean we are expected to expand the tgz archive?

Reply:
yep, done